shell/platform/android/io/flutter/embedding/engine/loader/FlutterLoader.java

```diff
@@ -42,6 +42,7 @@
   static final String VM_SNAPSHOT_DATA_KEY = "vm-snapshot-data";
   static final String ISOLATE_SNAPSHOT_DATA_KEY = "isolate-snapshot-data";
   static final String FLUTTER_ASSETS_DIR_KEY = "flutter-assets-dir";
+  static final String AUTOMATICALLY_REGISTER_PLUGINS_KEY = "automaticallyRegisterPlugins";
```

Comment:
these are for the C++ switches, but apparently they can also be set in the XML

Reply:
Yeah, I think that's the intention, such that internal project can turn this off in the internally shared manifest.xml

https://github.com/flutter/flutter/issues/72027#issuecomment-742104100